src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -155,7 +155,7 @@ const styles = StyleSheet.create({
   attachmentFileInfo: {
     fontSize: dimen.normalizeDimen(12),
     fontFamily: fonts.inter[400],
-    color: COLORS.gray
+    color: COLORS.gray500
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the attachmentFileInfo has been changed. As long as this new color fits within the design guidelines and improves readability, this change should be fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->